source/agora/node/FullNode.d

```diff
@@ -373,7 +374,10 @@ public class FullNode : API
     protected Clock getClock (TaskManager taskman)
     {
         // non-synchronizing clock (for now)
-        return new Clock(taskman, (out long time_offset) { return true; });
+        return new Clock(
+            (out long time_offset) { return true; },
+            (Duration duration, void delegate() cb) nothrow @safe
+                {()@trusted {this.taskman.setTimer(duration, cb, Periodic.Yes);}();});
```

Comment:
Since this is a one-liner function it's ok to just mark it entirely as `@trusted`:

```suggestion
            (Duration duration, void delegate() cb) nothrow @trusted
                { this.taskman.setTimer(duration, cb, Periodic.Yes); });
```